mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -192,7 +201,11 @@ final class OneVsRest(override val uid: String)
       val labelUDFWithNewMeta = labelUDF.as(labelColName, newLabelMeta)
       val trainingDataset = multiclassLabeled.withColumn(labelColName, labelUDFWithNewMeta)
       val classifier = getClassifier
-      classifier.fit(trainingDataset, classifier.labelCol -> labelColName)
+      val paramMap = new ParamMap()
+      paramMap.put(classifier.labelCol -> labelColName)
+      paramMap.put(classifier.featuresCol -> getFeaturesCol)
+      paramMap.put(classifier.predictionCol -> getPredictionCol)
```

Comment:
Is this saying that the base classifier should output "predictionCol"?  It probably should send output to a temp col, to be removed.